### ISSUE TITLE
Firebase file upload example won’t work involving fetch 

### ISSUE THREAD

**Issue by djaoui** [expo/expo#10526]
### Summary of Issue  
I m currently developing an App (not in production) and Firebase file upload feature using  Expo example worked fine on both iOS and Android on expo 38 before expo 39 upgrade. Not working on iOS (real device iPhone X with latest expo App) after upgrade but still fine on Android (emulator on PC Win10 with latest expo App)

### Environment - output of `expo diagnostics` & the platform(s) you're targeting

App in development in managed workflow
Error message “fetch error: network fail”

### Reproducible Demo

Exact same code taken from Expo Firebase file upload example from @EvanBacon 
Involves fetch() and blob()

**Comment by brentvatne** [comment]
see this thread: https://forums.expo.io/t/blob-stopped-working-in-expo-sdk-39/43516

tl;dr there was a regression in react-native@0.63.2, it has been resolved and will be released in a sdk 39 update soon, we will post more info to our blog and to twitter @expo when it's released. the thread provides some workarounds for some situations.

**Comment by djaoui** [comment]
Thank you so much @brentvatne for this and expo overall !!!
Looking forward to the update. Take care.
